### PR TITLE
Implement releases path and current path options

### PR DESCRIPTION
**Pull request by j1v3WKF**
This is related to this issue https://github.com/shipitjs/shipit-deploy/issues/91

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 node_modules/
+.idea
```
Comment (neoziro) [inline .gitignore]: Use your global .gitignore for this kind of thing, it's not specific to the project

File: tasks/deploy/publish.js [hunk 2]
```diff
@@ -25,7 +25,10 @@ module.exports = function (gruntOrShipit) {
     function updateSymbolicLink() {
       shipit.log('Publishing release "%s"', shipit.releasePath);
 
-      var relativeReleasePath = path.join('releases', shipit.releaseDirname);
+      if ( shipit.config.releasesPath == undefined )
```
Comment (neoziro) [inline tasks/deploy/publish.js]: Already done in init? I think we don't need it here.

Comment (j1v3WKF) [inline tasks/deploy/publish.js]: if the user customize a release.path, i think we must configure the symbolic link accordingly... but you think that there's a better way?

File: .gitignore (hunk 1 above)
Comment (j1v3WKF) [inline .gitignore]: it's tue i fix it on https://github.com/shipitjs/shipit-deploy/pull/99

**Pull request closed without merging**